src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -1405,8 +1405,8 @@ treatAsInline = Set.fromList
 label :: PandocMonad m => LP m ()
 label = do
   controlSeq "label"
-  t <- untokenize <$> braced
-  updateState $ \st -> st{ sLastLabel = Just t }
+  t <- braced
+  updateState $ \st -> st{ sLastLabel = Just $ untokenize t }
 
```

Comment:
I don't have a problem with this, but I'm just curious whether there's a non-aesthetic reason for removing the fmap here?

Reply:
Depending on the optimizer, it may be slightly faster (no promises, but it definitely won't be slower), for the same reason that `map (x . y)` is better than `map x . map y`.